kong/client_test.go

```diff
@@ -32,6 +32,19 @@ func TestKongStatus(T *testing.T) {
 	assert.NotNil(status)
 }
 
+func TestKongConfig(t *testing.T) {
+	RunWhenDBMode(t, "off")
+	assert := assert.New(t)
+
+	client, err := NewTestClient(nil, nil)
+	assert.NoError(err)
+	assert.NotNil(client)
+
+	config, err := client.Config(defaultCtx)
+	assert.NoError(err)
+	assert.NotNil(config)
```

Comment:
Can't we make some assertions about this config struct? This could

- serve a documentation purpose to show readers how to use it
- guard us against unforeseen failures which `NotNil()` wouldn't